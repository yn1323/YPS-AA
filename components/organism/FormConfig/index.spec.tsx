import React from 'react'
import Component from '.'
import { mountWithTheme } from '@test/TestWrapper'

describe(`${Component.name}`, () => {
  test(`${Component.name}-snapshot`, () => {
    const wrapper = mountWithTheme(<Component />)
    expect(wrapper).toMatchSnapshot()
    wrapper.setProps({ isLoggedIn: true })
    expect(wrapper).toMatchSnapshot()
    wrapper.setProps({ initialLogin: true })
    expect(wrapper).toMatchSnapshot()
  })
})
