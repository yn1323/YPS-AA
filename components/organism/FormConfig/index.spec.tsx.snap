// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormConfig-snapshot 1`] = `
<FormConfig>
  <Center>
    <Center__Container>
      <main
        className="Center__Container-sc-rjeqkd-0 ekYtfj"
      >
        <Center__CenterDiv>
          <div
            className="Center__CenterDiv-sc-rjeqkd-1 gxxXam"
          >
            <div>
              Title
            </div>
            <div>
              Description
            </div>
            <div>
              Form
            </div>
          </div>
        </Center__CenterDiv>
      </main>
    </Center__Container>
  </Center>
</FormConfig>
`;

exports[`FormConfig-snapshot 2`] = `
<FormConfig
  isLoggedIn={true}
>
  <Center>
    <Center__Container>
      <main
        className="Center__Container-sc-rjeqkd-0 ekYtfj"
      >
        <Center__CenterDiv>
          <div
            className="Center__CenterDiv-sc-rjeqkd-1 gxxXam"
          >
            <div>
              Title
            </div>
            <div>
              Description
            </div>
            <div>
              Form
            </div>
          </div>
        </Center__CenterDiv>
      </main>
    </Center__Container>
  </Center>
</FormConfig>
`;

exports[`FormConfig-snapshot 3`] = `
<FormConfig
  initialLogin={true}
  isLoggedIn={true}
>
  <Center>
    <Center__Container>
      <main
        className="Center__Container-sc-rjeqkd-0 ekYtfj"
      >
        <Center__CenterDiv>
          <div
            className="Center__CenterDiv-sc-rjeqkd-1 gxxXam"
          >
            <div>
              Title
            </div>
            <div>
              Description
            </div>
            <div>
              Form
            </div>
          </div>
        </Center__CenterDiv>
      </main>
    </Center__Container>
  </Center>
</FormConfig>
`;
